internal/balancergroup/balancergroup.go

```diff
@@ -305,11 +281,14 @@ func (bg *BalancerGroup) AddWithClientConn(id, balancerName string, cc balancer.
 	// Store data in static map, and then check to see if bg is started.
 	bg.outgoingMu.Lock()
 	defer bg.outgoingMu.Unlock()
+	if bg.outgoingClosed {
+		return fmt.Errorf("BalancerGroup already closed")
```

Comment:
Nit: There are currently only two places where the balancer group returns an error by itself. Can we add a "balancergroup" prefix to those two and this one, that way, we can also ensure that the error string is not capitalized. Thanks.

Reply:
Added the prefix and removed `BalancerGroup` here.